models/users.js

```diff
@@ -24,6 +24,10 @@ function deleteUser(userId) {
     let users = getUsers();
     const userToDelete = users.find((user) => userId === user.id);
 
+    if (!userToDelete) {
+        throw new Error(`No such user. `);
```

Comment:
You don't need to use ` here - you can use '.

```
`No such user. `
```
vs
```
'No such user.'
```

Do you know what's the difference between the two?

Reply:
Yes, it is being used for interpolation. I'll fix that.

Reply:
Oh, or if you use another type of quotes, for example like in words "can't"